examples/voting-poll/sample-ui/README.md

```diff
@@ -0,0 +1,50 @@
+# Sample UI Project - Voting App
+
+## Description
+
+This is a demo application built using Next.js 14 and npm. The application allows users to create polls and vote on them using their Stargazer DAG wallet. Users can only vote if they have balance in their wallet.
```

Comment:
```suggestion
This is a demo application built using Next.js 14 and npm. The application allows users to create polls and vote on them using their Stargazer wallet. Users can only vote if they have balance in their wallet.
```

We should just call it Stargazer wallet. It's actually being used as an L0 token wallet in this case.